src/analysis/inner_product_space/basic.lean

```diff
@@ -2273,4 +2286,33 @@ lemma is_self_adjoint.restrict_invariant {T : E →ₗ[𝕜] E} (hT : is_self_ad
   is_self_adjoint (T.restrict hV) :=
 λ v w, hT v w
 
+section complex
+
+variables {V : Type*}
+[inner_product_space ℂ V]
+
+/-- A linear operator on a complex inner product space is self-adjoint precisely when
+`⟪T v, v⟫_ℂ` is real for all v.-/
+lemma is_self_adjoint_iff_inner_map_self_real (T : V →ₗ[ℂ] V):
+  is_self_adjoint T ↔ ∀ (v : V), conj ⟪T v, v⟫_ℂ = ⟪T v, v⟫_ℂ :=
+begin
+  split,
+  { intros hT v,
+    apply is_self_adjoint.conj_inner_sym hT },
+  { intro h,
+    rw is_self_adjoint,
+    intros x y,
```

Comment:
```suggestion
  { intro h x y,
```
? (Untested!)

Reply:
I just tried it locally and it worked for me?

Reply:
Whoops, sorry to make you check in order to teach me this lesson, but I see now that `intros` is smarter than I gave it credit for.

Reply:
Yup, it's pretty smart :-)